filebeat/docs/inputs/input-common-file-options.asciidoc

```diff
@@ -397,3 +401,42 @@ file that hasn't been harvested for a longer period of time.
 This configuration option applies per input. You can use this option to
 indirectly set higher priorities on certain inputs by assigning a higher
 limit of harvesters.
+
+[float]
+===== `file_identity`
+
+Different `file_identity` methods can be configured to suit the
+environment you are collecting log messages from.
+
+
+*`inode_deviceid`*:: The default behaviour of {beatname_uc} is to differentiate
```

Comment:
fwiw, this codebase is inconsistent between `behavior` and `behaviour` (i think ~2:1)